tests/pytests/test_wildcard.py

```diff
@@ -4,168 +4,152 @@
-        # 55xx & x55x & xx55 - 555x - x555 
-        env.expect('ft.search', index_list[i], "w'*55*'", 'LIMIT', 0 , 0).equal([1120])
-        env.expect('ft.search', index_list[i], "w'*5?5*'", 'LIMIT', 0 , 0).equal([796])
-        env.expect('ft.search', index_list[i], "w'*5??5*'", 'LIMIT', 0 , 0).equal([400])
-        env.expect('ft.search', index_list[i], "w'?*5??5*?'", 'LIMIT', 0 , 0).equal([0])
-        # 555x & x555 - 5555
-        env.expect('ft.search', index_list[i], "w'*555*'", 'LIMIT', 0 , 0).equal([76])
-        env.expect('ft.search', index_list[i], "w'*o55*'", 'LIMIT', 0 , 0).equal([444])
-        env.expect('ft.search', index_list[i], "w'*oo55*'", 'LIMIT', 0 , 0).equal([333])
-        env.expect('ft.search', index_list[i], "w'*oo555*'", 'LIMIT', 0 , 0).equal([33])
-        env.expect('ft.search', index_list[i], "w'*55?5*'", 'LIMIT', 0 , 0).equal([40])
-        env.expect('ft.search', index_list[i], "w'*o?55*'", 'LIMIT', 0 , 0).equal([696])
-        env.expect('ft.search', index_list[i], "w'*oo?55*?'", 'LIMIT', 0 , 0).equal([470])
-        env.expect('ft.search', index_list[i], "w'*oo?555*'", 'LIMIT', 0 , 0).equal([47])
-        env.expect('ft.search', index_list[i], "w'*oo55?5*'", 'LIMIT', 0 , 0).equal([30])
-        env.expect('ft.search', index_list[i], "w'*oo?555?*'", 'LIMIT', 0 , 0).equal([20])
-
-        # # 23xx & x23x & xx23 - 2323
-        # env.expect('ft.search', index_list[i], '*23*', 'LIMIT', 0 , 0).equal([1196])
-        # # 234x & x234
-        # start = time.time()
-        # env.expect('ft.search', index_list[i], '*234*', 'LIMIT', 0 , 0).equal([80])
-        # print (time.time() - start)
-        # start = time.time()
-        # env.expect('ft.search', index_list[i], '*o23*', 'LIMIT', 0 , 0).equal([444])
-        # print (time.time() - start)
-        # env.expect('ft.search', index_list[i], '*oo23*', 'LIMIT', 0 , 0).equal([333])
-        # env.expect('ft.search', index_list[i], '*oo234*', 'LIMIT', 0 , 0).equal([33])
-
-        # suffix
-        env.expect('ft.search', index_list[i], "w'*oo234'", 'LIMIT', 0 , 0).equal([3])
-        env.expect('ft.search', index_list[i], "w'*234'", 'LIMIT', 0 , 0).equal([40])
-        env.expect('ft.search', index_list[i], "w'*13'", 'LIMIT', 0 , 0).equal([400])
-
-    # test timeout
-    env.expect('FT.CONFIG', 'set', 'TIMEOUT', 1).ok()
-    env.expect('FT.CONFIG', 'set', 'ON_TIMEOUT', 'RETURN').ok()
-    env.expect('ft.search', index_list[0], "w'foo*'", 'LIMIT', 0 , 0).error() \
-      .contains('Timeout limit was reached')
-    #env.expect('ft.search', index_list[1], 'foo*', 'LIMIT', 0 , 0).error() \
-    #  .contains('Timeout limit was reached')
-
-    env.expect('FT.CONFIG', 'set', 'ON_TIMEOUT', 'FAIL').ok()
-    env.expect('ft.search', index_list[0], "w'foo*'", 'LIMIT', 0 , 0).error() \
-      .contains('Timeout limit was reached')
-    #env.expect('ft.search', index_list[1], 'foo*', 'LIMIT', 0 , 0).error() \
-    #  .contains('Timeout limit was reached')
+  env.skipOnCluster()
```

Comment:
Can we try to adjust the test to also run on cluster?

Reply:
Done, as well as aggregate